Fixed D2k armament offsets and some cleanup

reaperrr opened the pull request.

reaperrr committed:
Tweak d2k armament offsets of some actors

reaperrr committed:
Fix d2k bazooka explosion

to match original.

reaperrr committed:
Remove duplicate d2k explosion sequences, increase explosion tick, fix deviator explosion offset

penev92 commented on mods/d2k/rules/structures.yaml:
```diff
@@ -593,7 +593,7 @@ large_gun_turret:
 	Armament:
 		Weapon: TowerMissile
-		LocalOffset: 256,384,768, 256,-384,768
+		LocalOffset: 256,0,768
```
What is the reason for this? This looks like it is removing some polish.
Same for the missile tank.

GraionDilach replied:
This one - haven't checked the missile tank yet - doesn't have burst on the weapon, which is why the swapping launchers back and forth is changed to a "fire between the two" look. Another polish could be halving missile damage and giving it Burst: 2 to maintain the image of using both launchers.

penev92 replied:
That would be great, but I guess it's out of scope here.

reaperrr replied:
Applies to missile tank as well.

We could also leave damage as it is, but still use Burst: 2 and make the BurstDelay identical to ReloadDelay, then the rate of fire would remain the same but it would alternate between left and right.

penev92 replied:
Alternating would be great, as long as we don't change the effective DPS of the unit (and don't make it fire 2 missiles at once).

reaperrr committed:
Use both firing offsets for missile tank and missile tower

Remove missile tank FireDelay. Fix Report of missile tank weapon. Fix and explosion animation of both missile weapons and use less audible explosion sound (had no sound at all in the original).

reaperrr committed:
Fix siege tank weapon explosion sound

reaperrr committed:
Remove incorrect anti-air capability from quad rocket

reaperrr commented:
Updated:
- Reverted the offset changes and made the missile weapons alternate between left and right.
- Fixed quad rockets (they were anti-ground only in the original).
- Corrected missile tank report and weapon explosion anim, also used a less noticable explosion sound (the original had none at all, but that felt wrong, too) and removed armament FireDelay.
- Corrected missile tower weapon explosion and sound to match missile tank.
- Corrected siege tank weapon explosion sound.

penev92 commented:
Looks good to me :+1: 
Thanks for all the cleanups.

pchote commented:
Looks fine to me, too. :+1:

pchote commented:
https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/7a79164cbdc008a98730522b6219bad36d412ee4
